client/src/components/ui/navigationbar/SideDrawer.svelte

```diff
@@ -56,11 +60,15 @@
                 <Button type={ButtonType.Primary}><Search color={IconColor.White} alt="Search specified tracking number." /></Button>
             </div>
         </header>
-        <a href="#/inbox" use:active><InboxIcon alt="Go to Inbox" />Inbox</a>
-        <a href="#/outbox" use:active><OutboxIcon alt="Go to Outbox" />Outbox</a>
-        <a href="#/dossier" use:active><Document alt="Go to Dossier" />Dossier</a>
+        {#if !localPermission || checkPerms(localPermission, Local.ViewInbox)}
```

Comment:
We can remove the extra check by making `localPermission` required.
```suggestion
        {#if checkPerms(localPermission, Local.ViewInbox)}
```